### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by vogella**
Adding a readme to document that the content of these repo has been merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+Moved to https://github.com/eclipse-pde/eclipse.pde
+===================================================
+
+The content of this repo has been moved to https://github.com/eclipse-pde/eclipse.pde by merging both master branches together.
+
+This repository is therefore obsolet for any new developments, it is still available to document the developments which went into oder releases via the maintenance branches.
```
Comment (HannesWell) [inline README.md:6]: `older`?

**Review by HannesWell — CHANGES_REQUESTED** [review]
Thanks Lars!
Just one suspected misspelling.

**Comment by vogella** [comment]
I triggered the change via the GH web UI. Do you know if I still can  edit the file or allow you to edit the change? Or do I have to pull this change locally to update it?

File: README.md (hunk 1 above)
Comment (HannesWell) [reply README.md:6]:
```suggestion
This repository is therefore obsolete for any new developments, it is still available to document the developments which went into older releases via the maintenance branches.
```

**Comment by HannesWell** [comment]
> I triggered the change via the GH web UI. Do you know if I still can edit the file or allow you to edit the change? Or do I have to pull this change locally to update it?

I think you can commit the suggested change from the UI (which probably adds a new commit) and then squash-and-merge all together in the end.

**Review by HannesWell — APPROVED** [review]
Looks good.
If we want to be super helpful, we could add the PR that merged this repo into o.e.pde.
But I'm not sure if this would really help readers later, especially since pde.build might evolve (away?) in the future.

**Comment by merks** [comment]
I noticed that the main page is quite out of date now...

https://github.com/eclipse-pde

Also, we might want to change the POM ids too:
```
  <artifactId>eclipse.pde.ui</artifactId>
```

**Comment by vogella** [comment]
@merks sounds fine to me, lets do this via new pull requests.

**Comment by HannesWell** [comment]
> I noticed that the main page is quite out of date now...
> 
> https://github.com/eclipse-pde

I think this is https://github.com/eclipse-pde/eclipse.pde/issues/12. Feel free to take over. :)

> Also, we might want to change the POM ids too:
> 
> ```
>   <artifactId>eclipse.pde.ui</artifactId>
> ```

Yes, I have already started to work on this. I plan to submit this this evening.

**Pull request merged**